draft-ietf-quic-recovery.md

```diff
@@ -477,6 +477,9 @@ limited by the endpoint's idle timeout.
 A sender computes its PTO timer every time an ack-eliciting packet is sent.
 When ack-eliciting packets are in-flight in multiple packet number spaces,
 the timer MUST be set for the packet number space with the earliest timeout.
+Even when there are ack-eliciting packets in-flight in multiple packet number
+spaces, the exponential increase in probe timeout occurs across all spaces to
```

Comment:
This interacts weirdly with the rest of this spec if I'm not coalescing, as I'm restricted to one packet per PTO. In the common case we are PTOing the server 1st flight, we get this behavior:
where P = srtt + 4 rttvar + max_ack_delay
 
time 0: send first flight
time P: send Initial
time 2P: send Handshake
time 4P: send 1RTT
time 9P: send Initial
time 18P: send Handshake

Maybe the lesson here is to coalesce, but we should understand what we're writing here.

Reply:
I removed the 1 packet limit when not coalescing when pointing out this existing exponential backoff behavior, so it should be:
 time 0: send first flight
 time P: send Initial and Handshake
 time 2P: Send 1RTT and Initial
 time 4P: send Handshake and 1RTT

Hopefully before 2P the server receives an ACK of something anyway, since that's seconds into the connection.

But we may want to tweak this as a result of the discard keys conclusion.

Reply:
I've updated the PR to no longer arm PTO for ApplicationData until the handshake is complete.  On the server side, it's pretty obvious you shouldn't arm it, since the server doesn't have receive keys to process 1-RTT ACKs.  I think this achieves what you originally might have hoped and solves the silliest case(server sending a 1-RTT PTO) above.

PTAL.